src/api/app/controllers/comments_controller.rb

```diff
@@ -2,7 +2,7 @@ class CommentsController < ApplicationController
   before_action :find_obj, only: [:index, :create]
 
   def index
-    @comments = @obj.comments.order(:id)
+    @comments = @obj.comments.preload(:user).order(:id)
```

Comment:
I don't think you need to enforce the preload. Just add an `include(:user)`. As soon as some chained method call triggers the query, the comments will be included / prefetched.

Reply:
the difference between includes and preload is marginal - if even existant in this case.

What we might want to use though is join (or eager_load)

Reply:
I can't see any difference between preload and includes. But it seems like includes is more known - so let's use this one